### PR TITLE
Deployer v7

### PR DESCRIPTION
**Pull request by antonmedv**
This PR will contain my work on next Deployer v7.

### CODE REVIEW

File: src/Console/WorkerCommand.php [hunk 1]
```diff
@@ -93,22 +77,40 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-        if ($task->shouldBePerformed($host)) {
-            $task->run(new Context($host, $input, $output));
-            $this->deployer->informer->endOnHost($hostname);
+        Storage::flush($this->host);
+    }
+
+    private function parseOptions(array $options)
+    {
+        foreach ($options as $option) {
+            list($name, $value) = explode('=', $option);
```
Comment (staabm) [inline src/Console/WorkerCommand.php]:
```suggestion
            list($name, $value) = explode('=', $option, 2);
```

**Comment by alexandrecruz** [comment]
@antonmedv do we have a roadmap for it?
Accepting help with the project?

**Comment by antonmedv** [comment]
Hi,

Yes, some sort of roadmap is on the way. I think I'll release the first alpha version after rebasing v7 to the master. PR can be merged after this.

**Comment by antonmedv** [comment]
Okay, I have plans to merge into master and continue to develop next version in master.

**Pull request merged**